rust/src/smb/detect.rs

```diff
@@ -156,11 +152,9 @@ pub extern "C" fn rs_smb_tx_get_dce_iface(state: &mut SMBState,
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn rs_smb_tx_get_ntlmssp_user(tx: &mut SMBTransaction,
-                                            buffer: *mut *const u8,
-                                            buffer_len: *mut u32)
-                                            -> u8
-{
+pub unsafe extern "C" fn rs_smb_tx_get_ntlmssp_user(
+    tx: &mut SMBTransaction, buffer: *mut *const u8, buffer_len: *mut u32,
+) -> u8 {
```

Comment:
Are these purely formatting changes to existing code? If yes, we usually prefer to see those in a separate commit, so its easier to visualize and review :)

Reply:
They are, I will have the rustfmt/clippy changes in a separate commit in the next PR.

Reply:
Happens to us all, thanks! :)